Dockerfile

```diff
@@ -5,8 +5,10 @@ COPY . /src
 
 WORKDIR /src
 
+RUN npm install -g gulp
```

Comment:
You don't need to install gulp globally. When you run `npm test`, npm will add any installed module's to the path, so it'll use the locally installed gulp.